src/RTCSession/RTCMediaHandler.js

```diff
@@ -378,6 +378,11 @@ RTCMediaHandler.prototype = {
       };
     } 
     logger.log("certificate : "+ExSIP.Utils.toString(certificate), this.session.ua);
+    if(!ExSIP.WebRTC.RTCPeerConnection.generateCertificate) {
```

Comment:
can you put that stub method into WebRTC.js so we keep all the stub methods in one place?